.github/workflows/ci-build.yaml

```diff
@@ -300,6 +298,8 @@ jobs:
         with:
           python-version: '3.x'
           cache: pip
+      - name: Install socat (needed by Kubernetes) and kit
+        run: sudo apt-get -y install socat && make kit
```

Comment:
Adding `make kit` isn't necessary, since `kit` is a prerequisite for `make start`, but it's noisy, and I figure it's better to avoid polluting the logs for `Start controller/API`